src/components/Button.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+export default class Button extends Component {
+  constructor(props) {
+    super(props);
+    this.props = props;
+  }
+
+  render() {
+    const { buttonName } = this.props;
+
+    return (
+      <p className="button">{buttonName}</p>
+    );
+  }
+}
+
+Button.defaultProps = {
+  buttonName: '',
+};
+
+Button.propTypes = {
+  buttonName: PropTypes.string,
+};
```

Comment:
OPTIONAL

- [ ] Since we do not have any case of a button without a name, it makes sense to make the `buttonName` props required, therefore, a default value is not required.
```JS
Button.propTypes = {
  buttonName: PropTypes.string.isRequired,
};

```

Reply:
It is good advice, thank you. Done.